src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -293,6 +293,9 @@ public static double Abs(Complex value)
 
         private static double Hypot(double a, double b)
         {
+#if NET7_0_OR_GREATER
+            return double.Hypot(a, b);
+#else
```

Comment:
We can just replace the entire impl with a direct call to `double.Hypot` since `System.Runtime.Numerics` only compiles for `$(NetCoreAppCurrent)` (and therefore only .NET 9

Several of these helpers, including `Log1P` can just be removed entirely and we can call the API on `double` (such as `LogP1`) directly instead

Reply:
Ah that makes things easier.

I see `Log1P` has some extra logic to mitigate loss of accuracy for when `x < 0.75`. just checking that is an OK change since `double.LogP1` is a more "naive" implementation?

Reply:
Ah right, I'd leave `Log1P` alone for now then, it's a backlog item to update the implementation to do the more accurate thing.